pkg/constants/constants.go

```diff
@@ -377,6 +377,7 @@ var (
 		StorageInitializerSourceUriInternalAnnotationKey,
 		"kubectl.kubernetes.io/last-applied-configuration",
 		"security.opendatahub.io/enable-auth",
+		"networking.knative.io/visibility",
```

Comment:
I think that there is no need to add it here, as this feature was not enabled before, and there is no need to keep backwards compatibility for upgrades.

@israel-hdez what do you think?